share/initialization/common

```diff
@@ -115,7 +115,8 @@ case $- in (*m*)
 esac
 
 # if the terminal supports color...
-if [ "$(tput colors 2>/dev/null || echo 0)" -ge 8 ]; then
+if [ $COLORTERM = "truecolor" ] || [ $COLORTERM = "24bit" ] || \
```

Comment:
This currently fails if COLORTERM is not defined.

Reply:
Try this

```suggestion
if [ "${COLORTERM-}" = "truecolor" ] || [ "${COLORTERM-}" = "24bit" ] || \
```